src/actions/noting/reset-note-barriers.js

```diff
@@ -31,18 +32,18 @@ module.exports = function resetNoteBarriers(focus, tagName = config.get('default
     //last note
     // This is necessarily complex (been through a few iterations) because
     // of Chrome's lack of flexibility when it comes to placing the caret.
-    var lastNote = noteSegments.slice(-1)[0].find((node)=> isNotWithinNote(node, tagName));
-    //find the first non-empty text node after the note
-    var adjacentTextNode = lastNote && lastNote.find(isNotEmpty);
-    var adjacentTextNodeContainsOnlyLink = adjacentTextNode && adjacentTextNode.vNode.text && !!adjacentTextNode.vNode.text.match(/^https?:\/\//);
-
-    if(adjacentTextNodeContainsOnlyLink){
-      adjacentTextNode.parent.addChild(new VText('\u200B'))
-    } else if (adjacentTextNode){
-      adjacentTextNode.vNode.text = '\u200B' + adjacentTextNode.vNode.text;
+    var lastnote = noteSegments.slice(-1)[0];
+    if (lastnote.next().text && /\u200B/g.test(lastnote.next().text)){
```

Comment:
Could we possibly standardise the zero width space character (as it put it in a global), because I have not idea was ```\u200B``` represents?